specifyweb/frontend/js_src/lib/components/WorkBench/hooks.ts

```diff
@@ -568,6 +568,16 @@ export function getHotHooks(wbView: WbView) {
           10
         );
     },
+    afterSelection: () => {
```

Comment:
Could you add one blank line before `afterSelection` to be consistent with other functions in this file?
Also, like you may see for other hooks in this file, you could please add a small comment explaining why this hook is needed?